src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -63,45 +63,21 @@
       (style/create-form-label "Select Entity")
       [:div {:style {:backgroundColor "#fff" :border (str "1px solid " (:line-gray style/colors))
                      :padding "1em" :marginBottom "0.5em"}}
-       (let [attribute-keys (apply
-                             union
-                             (map #(set (keys (% "attributes")))
-                                  (filter #(= (% "entityType") (:selected-entity-type @state))
-                                          (:entities props))))
-             types (:entity-types props)
-             root (:root-entity-type props)
-             ordered-types (if (contains? (set types) root)
-                             (cons root (remove #(= % root) types))
-                             types)]
-         [table/Table
-          {:key (:selected-entity-type @state)
-           :empty-message "No entities available."
-           :columns (concat
-                     [{:header "" :starting-width 40 :resizable? false :reorderable? false
-                       :sort-by :none :filter-by :none
-                       :content-renderer
-                       (fn [data]
-                         [:input {:type "radio"
-                                  :checked (identical? data (:selected-entity @state))
-                                  :onChange #(do (swap! state assoc :selected-entity data)
-                                                 ((:entity-selected props) (entity->id data)))}])}
-                      {:header "Entity Type" :starting-width 100}
-                      {:header "Entity Name" :starting-width 100}]
-                     (map (fn [k] {:header k :starting-width 100}) attribute-keys))
-           :filters (mapv (fn [key] {:text key :pred #(= key (% "entityType"))}) ordered-types)
-           :selected-filter-index (.indexOf (to-array ordered-types)
-                                            (:selected-entity-type @state))
-           :on-filter-change #(swap! state assoc
-                                     :selected-entity-type (nth ordered-types %))
-           :data (:entities props)
-           :->row (fn [m]
-                    (concat
-                     [m
-                      (m "entityType")
-                      (m "name")]
-                     (map (fn [k] (get-in m ["attributes" k])) attribute-keys)))}])]
+       [:div {:style {:marginBottom "1em" :fontSize "140%"}}
+        (str "Selected: "
+          (if-let [e (:selected-entity @state)]
+            (str (e "name") " (" (e "entityType") ")")
+            "None"))]
+       [table/EntityTable
+        {:entities (:entities props)
```

Comment:
Needs an attribute renderer? And maybe an assertion in the EntityTable when this is accidentally left out?